jib-gradle-plugin/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-bin.zip
```

Comment:
I think this enables us (developers) to potentially use 5.0 level APIs which then might mean we have to update our minimum gradle supported version in: `JibPlugin`

Can we keep a test that runs against `JibPlugin.GRADLE_MIN_VERSION` or just change that to `5.0` also. Changing it to minimum 5.0 might be annoying to users though.

Reply:
Good point. To my surprise, it is [currently not possible to specify the Gradle API dependency version](https://github.com/gradle/gradle/issues/1835).

I've restored the Gradle TestKit test to use the `JibPlugin.GRADLE_MIN_VERSION` (4.6). However, 4.6 just can't run on JDK 11, so I had to add `assumeTrue(Java 8)`. As long as we have a Java 8 build, we should be fine. The test was lacking applying the Jib plugin, so I also enhanced it to load the existing simple `build.gradle`:
```build.gradle
plugins {
    id 'java'
    id 'com.google.cloud.tools.jib'
}
```
Not sure if this will always catch any API usage greater than 4.6 though.

Reply:
BTW, although `GRADLE_MIN_VERSION` was 4.6, our build system was using 4.10-2, so we were developing against the 4.10-2 API anyway. I think it is not impossible that we already used API requiring >4.6.

Reply:
Seems like we don't need Gradle 5.0 if not targeting 11. Restored back to 4.10.2.